schema/property/holder.json

```diff
@@ -0,0 +1,189 @@
+                {
+                    "$ref": "../properties_directory/non-scalar/vibrational_spectrum.json"
+                },
+                {
+                    "$ref": "../properties_directory/non-scalar/phonon_dos.json"
+                },
+                {
+                    "$ref": "../properties_directory/non-scalar/potential_profile.json"
+                },
+                {
+                    "$ref": "../properties_directory/non-scalar/reaction_energy_profile.json"
+                },
+                {
+                    "$ref": "../properties_directory/non-scalar/density_of_states.json"
+                },
+                {
+                    "$ref": "../properties_directory/non-scalar/dielectric_tensor.json"
+                },
+                {
+                    "$ref": "../properties_directory/non-scalar/file_content.json"
+                },
+                {
+                    "$ref": "../properties_directory/non-scalar/hubbard_u.json"
+                },
+                {
+                    "$ref": "../properties_directory/non-scalar/hubbard_v.json"
+                },
+                {
+                    "$ref": "../properties_directory/non-scalar/hubbard_v_nn.json"
+                },
+                {
+                    "$ref": "../properties_directory/non-scalar/average_potential_profile.json"
+                },
+                {
+                    "$ref": "../properties_directory/non-scalar/charge_density_profile.json"
+                },
+                {
+                    "$ref": "../properties_directory/non-scalar/workflow.json"
+                },
+                {
+                    "$ref": "../properties_directory/structural/magnetic_moments.json"
+                },
+                {
+                    "$ref": "../properties_directory/structural/atomic_forces.json"
+                },
+                {
+                    "$ref": "../properties_directory/structural/basis/atomic_constraints_property.json"
+                },
+                {
+                    "$ref": "../properties_directory/structural/patterns/functional_group.json"
+                },
+                {
+                    "$ref": "../properties_directory/structural/patterns/ring.json"
+                },
+                {
+                    "$ref": "../properties_directory/structural/patterns/special_bond.json"
+                },
+                {
+                    "$ref": "../properties_directory/workflow/convergence/electronic.json"
+                },
+                {
+                    "$ref": "../properties_directory/workflow/convergence/ionic.json"
+                },
+                {
+                    "$ref": "../properties_directory/non-scalar/is_relaxed.json"
+                },
+                {
+                    "$ref": "../properties_directory/non-scalar/final_structure.json"
+                }
+            ]
+        },
+        "source": {
```

Comment:
We could reuse source for meta and proto holders

Reply:
I'm not sure we need this structure for Meta and Proto holders, as they contain an empty `source`:
```
"source" : {
        "info" : {

        },
        "type" : "exabyte"
    }
```
And holder has:
```
"source" : {
        "type" : "exabyte",
        "info" : {
            "jobId" : "HKPDxdJgGX5FrXLCy",
            "unitId" : "f23e15fa1304b005d942d64c-2ba70021fabbc28cbfb7cee0"
        }
    }
```